packages/terra-button-group/tests/nightwatch/ButtonGroupTestRoutes.jsx

```diff
@@ -3,7 +3,7 @@
 import React from 'react';
 import { Route } from 'react-router';
 import ButtonGroupTests from './ButtonGroupTests';
-import { TextButtonGroup, IconButtonGroup, LongTextButtonGroup } from './ButtonGroupTextAndIcon';
+// import { TextButtonGroup, IconButtonGroup, LongTextButtonGroup } from './ButtonGroupTextAndIcon';
```

Comment:
Should we just remove this or will this be added back later?

Reply:
Eventually this file will be deleted. See https://github.com/cerner/terra-core/issues/1255. However, removing all of these Num of Components * 2 added a ridiculous about of additional files changes in this PR that @bjankord asked to fix this issue in a new PR